src/sentry/static/sentry/app/views/performance/transactionSummary/latencyChart.tsx

```diff
@@ -122,18 +74,11 @@ class LatencyChart extends AsyncComponent<Props, State> {
     this.setState({zoomError: true});
   };
 
-  get bucketWidth() {
-    if (this.state.chartData === null) {
-      return 0;
-    }
+  bucketWidth(data: HistogramData[]) {
```

Comment:
We should prob make histogram utils at some point to abstract this out of a few places, but we can clean that up later.